ghost/ghost-app/ingress-ghost.yaml

```diff
@@ -14,3 +14,5 @@ spec:
       - backend:
           serviceName: ghost-app
           servicePort: 80
```

Comment:
You'll also need to update the structure of `backend` since you updated the apiVersion from v1beta1 to v1.

For example:
```
backend:
  service:
    name: ghost-app
    port:
      number: 80
```

Reply:
Thank you :), I realized during the testing :) and reverted it for now.